### ISSUE TITLE
Previous version warning is misleading

### ISSUE THREAD

**Issue by tnozicka** [scylladb/sphinx-scylladb-theme#925]
We have docs for (unreleased) v1.11, and released v1.10. When I open the v1.11 branch I get message saying:
```
You're viewing documentation for a previous version of Scylla Operator. [Switch to the latest stable version.](https://operator.docs.scylladb.com/stable/index.html)
```
and a link that leads me to v1.10 docs. This is misleading since v1.11 isn't `previous` to v1.10, it's just not released yet.

It would be great to rephrase it to avoid this confusion. Maybe something like
```
You're not viewing the latest stable documentation of Scylla Operator. [Switch to the latest stable version.](https://operator.docs.scylladb.com/stable/index.html)
```
![Screenshot from 2023-10-20 11-54-32](https://github.com/scylladb/sphinx-scylladb-theme/assets/11869554/c7d49c7f-847e-466b-b53f-00f5739f6c26)

**Comment by dgarcia360** [comment]
Hi @tnozicka,

Now it is possible to define versions as "unstable": https://sphinx-theme.scylladb.com/stable/configuration/multiversion.html#defining-unstable-versions

<img width="642" alt="image" src="https://github.com/scylladb/sphinx-scylladb-theme/assets/9107969/292a6531-1321-4a6f-b48c-03a76929a29e">

Does that work for you?

**Comment by tnozicka** [comment]
we can do that, thanks
